docs/user_guide/workloads/run_spark.md

```diff
@@ -25,12 +25,86 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+## Deploy Spark job with Spark Operator and Helm
+
+:::note
+Pre-requisites:
+- This tutorial assumes YuniKorn is [installed](../../get_started/get_started.md) under the namespace `yunikorn`
+- Use spark-operator version >= 2.0 to enable support for YuniKorn gang scheduling
+:::
+
+:::warning
```

Comment:
Consider changing it to `note` instead of `warning`. Spending a few minutes on installation doesn't seem to warrant a warning

Reply:
Thank you for pointing it out. I actually forget to delete it. It's not a problem this version of steps.